pkg/operator/operator.go

```diff
@@ -221,17 +226,19 @@ func New(config operatorconfig.Config, kubeConfig *rest.Config) (*Operator, erro
 // synchronously until a message is received on the stop channel.
 // TODO: Move the default IngressController logic elsewhere.
 func (o *Operator) Start(ctx context.Context) error {
-	// Periodicaly ensure the default controller exists.
-	go wait.Until(func() {
-		if !o.manager.GetCache().WaitForCacheSync(ctx) {
-			log.Error(nil, "failed to sync cache before ensuring default ingresscontroller")
-			return
-		}
-		err := o.ensureDefaultIngressController()
-		if err != nil {
-			log.Error(err, "failed to ensure default ingresscontroller")
-		}
-	}, 1*time.Minute, ctx.Done())
+	if os.Getenv(IgnoreAutoCreateDefaultIngressControllerEnvVar) != "true" {
```

Comment:
Regardless of where this logic ends up per my last comment about `OperatorConfig.config`, you should use `strconv.ParseBool(VAR)`

Reply:
fixed